### PR TITLE
Add TPS62132 and TPS62133 DC/DC Converters

### PR DESCRIPTION
**Pull request by JMW95**
TPS62132 3.3V 3A fixed output step-down converter
TPS62133 5V 3A fixed output step-down converter

### CODE REVIEW

File: lib/ic/power/tps62132.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+description: 3.3Vout 3A, fixed, Buck Converter
+footprint: agg:QFN-16-EP-TI
+name: TPS62132
+ordercodes:
+- [Farnell, '2492468']
+pins:
+- - - [PVIN, 11, pwrin]
+    - [PVIN, 12, pwrin]
+    - [AVIN, 10, pwrin]
+    - [EN, 13, pwrin]
```
Comment (adamgreig) [inline lib/ic/power/tps62132.yaml]: EN should be an input rather than pwrin?

File: lib/ic/power/tps62132.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  - - [SS/TR, 9, passive]
```
Comment (adamgreig) [inline lib/ic/power/tps62132.yaml]: SS/TR should be input rather than passive?

File: lib/ic/power/tps62132.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  - - [DEF, 8, in]
+    - [FB, 5, passive]
```
Comment (adamgreig) [inline lib/ic/power/tps62132.yaml]: FB should be input rather than passive?

File: lib/ic/power/tps62132.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    - [AGND, 6, pwrin]
+    - [PGND, 15, pwrin]
+    - [PGND, 16, pwrin]
+    - [EP, EP, pwrin]
+- - - [SW, 1, pwrout]
```
Comment (adamgreig) [inline lib/ic/power/tps62132.yaml]: given as SW always connects to an inductor it'l never get seen as a pwrout - probably best to have them all passive?

**Comment by JMW95** [comment]
These changes are worth making to the existing TPS62152 part as well
Should I make a new PR for that, or change it here?

**Comment by adamgreig** [comment]
Good point, change it here please.

**Pull request merged**